syntax = "proto3";

service Times {
    rpc GetShowtimes(EmptyTimes) returns (stream TimesData) {}
    rpc GetShowtimesByDate(TimesDate) returns (TimesData) {}
}

message TimesDate {
    string date = 1;
}

message TimesData {
    string date = 1;
    repeated string timesMovies = 2;
}

message EmptyTimes {
}
